### PR TITLE
Removed t3_int isr

### LINKED ISSUE #133: Timer 3 use overflow bit
Timer 3 should use it's own overflow bit to track for timer overflows instead of checking this manually

### PR DESCRIPTION
**Pull request by damosvil**
Removed t3 interrupt service routine to avoid wasting necessary CPU cycles

To be merged after https://github.com/bird-sanctuary/bluejay/tree/feature/dynamic_pwm

### CODE REVIEW

File: src/Modules/Isrs.asm [hunk 1]
```diff
@@ -609,22 +609,8 @@ t2_int_exit:
 ;
 ; Timer3 interrupt routine
 ;
```
Comment (stylesuxx) [inline src/Modules/Isrs.asm:611]: The task of timer 3 is still commutation timing though - I think we can keep the "Tasks" documentation here, no?

File: src/Modules/Timing.asm [hunk 2]
```diff
@@ -691,8 +683,6 @@ comp_read_wrong_startup_jump:
     sjmp comp_check_timeout             ; Continue to look for good ones
 
 comp_read_wrong_extend_timeout:
-    clr  Flag_Demag_Detected            ; Clear demag detected flag
```
Comment (stylesuxx) [inline src/Modules/Timing.asm:694]: Why is it no longer necessary to clear the demag detection flag?

Comment (damosvil) [reply src/Modules/Timing.asm:694]: Good catch

File: src/Modules/Isrs.asm (hunk 1 above)
Comment (damosvil) [reply src/Modules/Isrs.asm:611]: It is explained at Bluejay.asm line 43

**Review by stylesuxx — APPROVED** [review]
Looking good to me!

Let's merge the other PR first and I'll merge this one into the v0.21 branch afterwards.

File: src/Modules/Timing.asm (hunk 2 above)
Comment (damosvil) [reply src/Modules/Timing.asm:694]: Fixed

Comment (damosvil) [reply src/Modules/Timing.asm:694]: Fixed

**Pull request closed without merging**